tests/worlds/mazeworld/test_weather.py

```diff
@@ -0,0 +1,132 @@
+"""
+Test that weather options can be configured both from scenario and programmatically
+"""
+import os
+import uuid
+from typing import List, Tuple
```

Comment:
Previously we haven't used type hints since they were introduced in python 3.5 and we targeted lower python version. However, [it seems <3.5 has now gone EOL](https://devguide.python.org/#status-of-python-branches), so we could start using them if we decided to drop support and enforce a minimum python version of 3.5. We should have this discussion